File: 단계별로_풀어보기/1. 입출력과 사칙연산/6_10869.py
#
# 1-5. 10869 사칙연산
#
# 입력
#   A, B: 10,000 이하의 자연수
#
# 출력
#   A + B
#   A - B
#   A * B
#   A / B (몫만 출력한다.)
#   A % B
#   각각 한 줄로 출력한다.
#

import sys
import math

input = sys.stdin.readline
output = sys.stdout.write

A, B = map(int, input().split())

output(f"{A + B}\n")
output(f"{A - B}\n")
output(f"{A * B}\n")
output(f"{math.floor(A / B)}\n")
output(f"{A % B}\n")

File: 단계별로_풀어보기/1. 입출력과 사칙연산/6_10869.js
/**
 * 1-5. 10869 사칙연산
 *
 * 입력
 *   A, B : 10,000 이하의 자연수
 *
 * 출력
 *   A + B
 *   A - B
 *   A * B
 *   A / B (몫만)
 *   A % B
 *   를 각각 한 줄씩 출력한다.
 */

const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().split(' ')
let [A, B] = input
A = parseInt(A)
B = parseInt(B)

const resultSum = (A + B).toString()
const resultSub = (A - B).toString()
const resultMul = (A * B).toString()
const resultDiv = Math.floor(A / B).toString() // 몫만 출력한다.
const resultMod = (A % B).toString()

const resultArr = [resultSum, resultSub, resultMul, resultDiv, resultMod]

process.stdout.write(resultArr.join('\n'))
